pkg/cmd/p2p/p2p.go

```
@@ -19,6 +19,9 @@ func NewP2PCmd(cfg *config.Config) (*cobra.Command, error) {
 			return nil
 		},
 	}
+	if cfg.DryRun {
+		panic("Cannot dry-run p2p commands")
```

Why not return an error?